examples/20171108_Pasadena/run_topoflux_example.py

```diff
@@ -1,14 +0,0 @@
-from isofit.core.isofit import Isofit
```

Comment:
Moved to `isofit/test/test_examples.py`. I don't see a reason to maintain multiple versions of the configuration.